test/util/util.go

```diff
@@ -795,3 +796,30 @@ func ExpectLocalQueuesToBeActive(ctx context.Context, c client.Client, lqs ...*k
 		}
 	}, Timeout, Interval).Should(gomega.Succeed())
 }
+
+func CreateNodes(ctx context.Context, c client.Client, nodes []corev1.Node) {
+	for _, node := range nodes {
+		// Create a node
+		gomega.ExpectWithOffset(1, c.Create(ctx, &node)).Should(gomega.Succeed())
+		// Update status based on the object
+		createdNode := &corev1.Node{}
+		gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
+			g.Expect(c.Get(ctx, client.ObjectKeyFromObject(&node), createdNode)).Should(gomega.Succeed())
+			createdNode.Status = node.Status
+			g.Expect(c.Status().Update(ctx, createdNode)).Should(gomega.Succeed())
+		}, Timeout, Interval).Should(gomega.Succeed())
+		// Removes the taint if the node is Ready
+		gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
+			if !utiltas.IsNodeStatusConditionTrue(node.Status.Conditions, corev1.NodeReady) {
```

Comment:
I think this should be the other way round - we filter if Ready

Reply:
Yes, you right. Thanks!